src/coprocessor/endpoint.rs

```diff
@@ -555,8 +563,7 @@ fn err_resp(e: Error, metrics: &mut BasicLocalMetrics, request_max_handle_secs:
             "lock"
         }
         Error::Outdated(deadline, now, scan_tag) => {
-            let elapsed =
-                now.duration_since(deadline) + Duration::from_secs(request_max_handle_secs);
+            let elapsed = now.duration_since(deadline) + request_max_handle_duration;
```

Comment:
Could we store `start_time` and `now` in `Error::Outdated` to avoid pass `request_max_handle_duration` here?